docs/deployment/considerations_production.md

```diff
@@ -0,0 +1,104 @@
+
+
+## AOAI Instances - Redundancy and Reliability
+
+**Consideration:** To enhance system reliability and ensure business continuity, consider implementing multiple instances of your Azure OpenAI (AOAI).
+
+**Recommendation:** You could deploy AOAI instances across multiple Azure regions for geographic redundancy. Leverage Azure Traffic Manager for global distribution and failover capabilities, ensuring uninterrupted service even in the event of a regional failure. Learn more about potential strategies [here](https://techcommunity.microsoft.com/t5/ai-azure-ai-services-blog/azure-openai-architecture-patterns-and-implementation-steps/ba-p/3979934).
+
+
+## GPT Model - Throttling
+
+**Consideration:** Throttling is essential to control the rate at which requests are sent to the GPT model, preventing overload and maintaining performance.
+
+**Recommendation**: Microsoft offers Provisioned Throughput options, allowing you to specify the maximum request rate for your GPT model. Define appropriate throughput limits based on the model's capacity and resource availability. Adjust throttling settings as needed to ensure optimal performance without risking service degradation. Learn more [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/provisioned-throughput).
+
+
+## Monitoring And Alerting
+
+**Consideration:** Proactively monitoring the IA environment is crucial for identifying performance issues, bottlenecks, and potential security threats.
+
+**Recommendation:** Leverage Azure Monitor to gain insights into the performance and health of your entire IA ecosystem. Set up custom alerts based on key performance indicators and metrics. Utilize Azure Log Analytics for in-depth analysis and troubleshooting. Integrate Azure Security Center to monitor and respond to security threats in real-time.
+Learn more [here](https://azure.microsoft.com/en-us/products/monitor/?ef_id=_k_2bb24bd93ec91aeba1fe2e4c90190298_k_&OCID=AIDcmm5edswduu_SEM__k_2bb24bd93ec91aeba1fe2e4c90190298_k_&msclkid=2bb24bd93ec91aeba1fe2e4c90190298).
+
+## Document Ingestion Limitations
+
+**Consideration:** This accelerator uses Azure Document Intelligence for PDF's and Unstructured.io for document processing. These tools have their own limitations when it comes to document sizes it can handle. The page limit for PDFs on Document Intelligence in Azure is 2000 pages. However, with a free tier subscription, only the first two pages are processed. The maximum document size to process with Azure AI Document Intelligence must not exceed 20 MB. The file size for analyzing documents must be less than 500 MB for paid (S0) tier and 4 MB for free (F0) tier. Learn more [here](https://learn.microsoft.com/en-us/legal/cognitive-services/document-intelligence/characteristics-and-limitations).
```

Comment:
I am cautious about listing specific limitations here as they will soon be out of date. I would rather note the limitations then send a user to the official documents link rather than have hard coded values that will be wrong shortly. IMO

Reply:
ah, that's a great point I should have thought about. I have updated!